koku/masu/database/trino_sql/verify/managed_ocp_on_cloud_tables.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+    count_1 = count_2 AS counts_match
+FROM
+    (
+        SELECT COUNT(*) AS count_1
+        FROM hive.{{schema | sqlsafe}}.{{managed_table | sqlsafe}} as managed_ocpaws
+        WHERE managed_ocpaws.source = {{aws_source_uuid}}
+        AND managed_ocpaws.ocp_source = {{ocp_source_uuid}}
```

Comment:
I will likely remove this filter condition, I think it might create false positives.